.agents/skills/release-validation/scripts/collect-release-inventory.py

```
@@ -0,0 +1,233 @@
+    if not fields:
+        return []
+    field_names = ("sha", "authored_at", "author", "author_email", "subject", "body")
+    if len(fields) % len(field_names) != 0:
+        raise RuntimeError("could not parse NUL-delimited git log records")
+
+    commits = []
+    for offset in range(0, len(fields), len(field_names)):
+        commit_fields = fields[offset : offset + len(field_names)]
+        commit_fields[-1] = commit_fields[-1].rstrip("\n")
+        commits.append(
+            dict(zip(field_names, commit_fields))
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)